### PR TITLE
Do not load 2fa admin settings async

### PR DESCRIPTION
**Pull request by rullzer**
This makes sure the 2fa state is loaded directly instead of fireing off an async request showing a little spinner.

TODO:
- [x] use initial state
- ~~Save data via store?~~

### CODE REVIEW

**Comment by ChristophWurst** [comment]
You might want to use your new API from https://github.com/nextcloud/server/pull/13646 once it's merged ;)

**Comment by ChristophWurst** [comment]
>     * Save data via store?

IMO unrelated -> setting to review

**Comment by rullzer** [comment]
Lets do this then. Thnx @ChristophWurst

**Review by ChristophWurst — APPROVED** [review]
:+1: for @rullzer's part

File: settings/src/components/AdminTwoFactor.vue [hunk 1]
```diff
@@ -77,33 +77,24 @@
...
+		computed: {
+			state: function() {
+				return  {
+					enforced: this.$store.state.enforced,
+					enforcedGroups: this.$store.state.enforcedGroups,
+					excludedGroups: this.$store.state.excludedGroups,
+				}
+			}
+		}
+		,mounted () {
```
Comment (MorrisJobke) [inline settings/src/components/AdminTwoFactor.vue]: Could you move the `,` to the line before?

**Review by MorrisJobke — APPROVED** [review]
Looks good and works (beside my little nitpick)

**Pull request merged**